plasma_framework/contracts/src/exits/payment/PaymentExitDataModel.sol

```diff
@@ -1,9 +1,21 @@
 pragma solidity ^0.5.0;
 
+/**
```

Comment:
Could we add a sample data for `StandardExit` and `InFlightExit` in json format so the context can be better understood?

Reply:
hmm sample data of InFlightExit would be pretty long though, what is the main part you'd like to have extra clarity?

```
{ exitStartTimestamp: 1569561351,
  exitMap: 80,
  position: 1000000000000,
  bondOwner: '0xf17f52151EbEF6C7334FAD080c5704D77216b732',
  bondSize: '37000000000000000',
  oldestCompetitorPosition: 0,
  inputs:
   [ { outputId:
        '0x28e655b6039b1b0b6b0bb2d9e73688248f5982090b328b1eb96d8f159b38a594',
       exitTarget: '0xC5fdf4076b8F3A5357c5E395ab970B5B54098Fef',
       token: '0x0000000000000000000000000000000000000000',
       amount: 667,
       piggybackBondSize: '28000000000000000' },
     { outputId:
        '0x550709e8d5a1bb24d69245e0d566252921907fcfbb233e0c29055373faff48d9',
       exitTarget: '0x821aEa9a577a9b44299B9c15c88cf3087F3b5544',
       token: '0x0000000000000000000000000000000000000000',
       amount: 667,
       piggybackBondSize: '28000000000000000' },
     { outputId:
        '0xd86a4212a1ff82b79ef3aedf737f714eef1ced10e8a014dcd60540abfe0d90d7',
       exitTarget: '0x0d1d4e623D10F9FBA5Db95830F7d3839406C6AF2',
       token: '0xFB88dE099e13c3ED21F80a7a1E49f8CAEcF10df6',
       amount: 667,
       piggybackBondSize: '28000000000000000' },
     { outputId:
        '0x088437a6f5532b5c550574551f6230150ac496af81eb27938bef5a5802be4a73',
       exitTarget: '0x0000000000000000000000000000000000000000',
       token: '0x0000000000000000000000000000000000000000',
       amount: 0,
       piggybackBondSize: 0 } ],
  outputs:
   [ { outputId:
        '0x2e6f452f1c2dfc17151c92a5b017f1cd6f64e2ea5aea7db74c6adea23b30de32',
       exitTarget: '0x2932b7A2355D6fecc4b5c0B6BD44cC31df247a2e',
       token: '0x0000000000000000000000000000000000000000',
       amount: 666,
       piggybackBondSize: '28000000000000000' },
     { outputId:
        '0xfda370418469bd46ae3f6cfce2cf66cf737e538c9e493824ce5fafa0fa3e10be',
       exitTarget: '0x2191eF87E392377ec08E7c08Eb105Ef5448eCED5',
       token: '0x0000000000000000000000000000000000000000',
       amount: 666,
       piggybackBondSize: '28000000000000000' },
     { outputId:
        '0xe4970c590baf7b5afcdd09f1b061230696edd2a9ec1220437d9fd628be4285f8',
       exitTarget: '0x0F4F2Ac550A1b4e2280d04c21cEa7EBD822934b5',
       token: '0xFB88dE099e13c3ED21F80a7a1E49f8CAEcF10df6',
       amount: 666,
       piggybackBondSize: '28000000000000000' },
     { outputId:
        '0x088437a6f5532b5c550574551f6230150ac496af81eb27938bef5a5802be4a73',
       exitTarget: '0x0000000000000000000000000000000000000000',
       token: '0x0000000000000000000000000000000000000000',
       amount: 0,
       piggybackBondSize: 0 } ] }
```

Reply:
Never mind, I can generate what I am looking for with omg-js.